### ISSUE TITLE
Time Management and Random Weather

### ISSUE THREAD

**Issue by DEL-J** [Diffusion9/Sandbox.Altis#29]
# Environment

## Time Management
Leaders need the ability to, without an administrator, control when they conduct operations. I propose the most immersive way to do this is to allow the highest ranked player on the server the ability to "rest" to skip time forward to their desired time of operation. May skip forward at increments of...?

When skipping time, there is not guarantee that AI will not attack players and spoil their operation.

## Random Weather
Arma's vanilla weather patterns are as predictable as they are awkward. We can do better. We can do fog, waves, wind, rain, and full blown storms randomly and in keeping with the appropriate parameters.

**Comment by Diffusion9** [comment]
Could make time management granular; have multiple options to "rest" or "wait" for a certain time, and certain conditions. Now that I'm more comfortable with dialogs, we could have a dialog appear with options for different times and weather conditions to "wait" for (wait would be simulated).

Will investigate better randomizing the weather. AFAIK the weather on Altis \ Stratis resembles their counterparts, so we could definitely more heavily randomize the conditions. We will have to formulate a list of conditions, minimums and extremes.

**Comment by DEL-J** [comment]
Yeah! This is what I was hopeful to hear! A dialog for a ranking individual that would let them just WAIT a set time for preferred conditions without input from an administrator.
